account_check_deposit/security/check_deposit_security.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+    Copyright (C) 2014-2019 Akretion (http://www.akretion.com/)
+    @author Alexis de Lattre <alexis.delattre@akretion.com>
+    The licence is in the file __manifest__.py
+-->
+<odoo noupdate="1">
+    <record id="check_deposit_rule" model="ir.rule">
+        <field name="name">Check Deposit multi-company</field>
+        <field name="model_id" ref="model_account_check_deposit" />
+        <field
+            name="domain_force"
+        >['|', ('company_id', '=', False), ('company_id', 'child_of', [user.company_id.id])]</field>
```

Comment:
```suggestion
        <field name="domain_force">['|', ('company_id', '=', False), ('company_id', 'in', company_ids)]</field>
```

https://www.odoo.com/documentation/13.0/howtos/company.html#security-rules

Reply:
Thank you for review. I edited this code.